### PR TITLE
kafka removal

### PR DESCRIPTION
**Pull request by ivukotic**
simple change

### CODE REVIEW

File: servicex/cache.py [hunk 1]
```diff
@@ -140,6 +138,11 @@ def __init__(self, cache_path: Path, ignore_cache: bool = False):
         self._path = cache_path
         self._ignore_cache = ignore_cache
 
+        # old windows versions have a limit of file path of 260 chars.
```
Comment (BenGalewsky) [inline servicex/cache.py:141]: What does this have to do with Kafka?

Comment (ivukotic) [reply servicex/cache.py:141]: it does not, but the test was failing so I fixed it.

**Review by BenGalewsky — APPROVED** [review]
Thanks for fixing the tests
